extension/intents/search/search.js

```diff
@@ -125,6 +125,59 @@ this.intents.search = (function() {
     }, CARD_POLL_INTERVAL);
   }
 
+  async function moveResult(context, step) {
+    stopCardPoll();
+    const { tabId, searchInfo } = await getSearchInfo();
+    if (!searchInfo) {
+      const e = new Error("No search made");
+      e.displayMessage = "You haven't made a search";
+      throw e;
+    }
+    if ((searchInfo.index >= searchInfo.searchResults.length - 1 && step > 0) ||
+    (searchInfo.index <= 0 && step < 0)) {
+      const tabId = await openSearchTab();
+      await context.makeTabActive(tabId);
+      return;
+    }
+    searchInfo.index += step;
```

Comment:
This should probably test that `searchInfo.index + step >= 0`, and give an error if it's not.

Reply:
I'll test that condition before increment line. What error message do you expect?.

I need some step by step guidance in order to update this PR with changes requested. I dont want to create another "rebase" mess.

Reply:
I guess the error can be "No previous search result" (using the whole `exc = new Error("No previous search result"); exc.displayMessage = "No previous search result"; throw exc` pattern)

You can just push a new commit. I'll squash it in GitHub when it lands. 

Another easy way to revise a commit is `git commit -a --amend` (this adds your changes to the previous commit) and then `git push -f`

Reply:
Thanks!